### PR TITLE
Troubleshooting tips for Solr.

### PR DESCRIPTION
**Pull request by damienmckenna**
This provides some tips for troubleshooting Solr.

### CODE REVIEW

File: docker-compose-services/solr-5/README.md [hunk 1]
```diff
@@ -45,4 +45,59 @@ DDEV has a note that it will clean up volumes that match the service name. Since
...
+
+## Troubleshooting
+
+If the Solr service is not available to the client project (e.g. Drupal) using
+the expected URL format, the Solr index may not have been created correctly. It
+is possible to access Solr directly from the host OS by loading its full URL,
+e.g. `http://myproject.ddev.site:8983`.
+
+* Make note that the URL uses the HTTP protocol, not HTTPS.
+* Safari may insist in loading the URL as HTTPS, e.g,
```
Comment (rfay) [inline docker-compose-services/solr-5/README.md:85]: Just add HTTPS_EXPOSE=8984 like the [core example has](https://github.com/drud/ddev/blob/master/pkg/servicetest/testdata/TestServices/docker-compose.solr.yaml) and then use `https://myproject.ddev.site:8984` everywhere and this problem goes away.

**Review by rfay — CHANGES_REQUESTED** [review]
Might as well just solve the http/https redirection problem with HTTPS_EXPOSE.

File: docker-compose-services/solr-5/README.md (hunk 1 above)
Comment (damienmckenna) [reply docker-compose-services/solr-5/README.md]: Good call, thank you. I've updated the README files to mention this.

**Comment by rfay** [comment]
I was saying just add the HTTPS_EXPOSE to the docker-compose.solr.yaml so you can leave that whole section out. Then change references to the web UI to use https.

**Review by rfay — APPROVED** [review]
Thanks for all the careful thinking here. Rebased and added a commit to add the HTTPS_EXPOSE to solr recipes that were missing it.

**Comment by damienmckenna** [comment]
Thanks for the refinements, @rfay!

**Pull request merged**